MetaMask/metamask-extension
# feat: add eth native icon for zora network

## **Description**

The zora network uses ETH as its native token, so it can have an ETH icon.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29257?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Add zora network
2. Verify native eth asset has eth icon

## **Screenshots/Recordings**



### **Before**
<img width="156" alt="Screenshot 2024-12-16 at 2 55 29 PM" src="https://github.com/user-attachments/assets/897a1d93-46c1-4d27-af4e-26c2fb41202f" />

### **After**

<img width="157" alt="Screenshot 2024-12-16 at 2 55 08 PM" src="https://github.com/user-attachments/assets/6e7b7d5a-4849-41d7-9080-ff5ac15913b3" />


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
